### ISSUE TITLE
Multiple controllers input?

### ISSUE THREAD

**Issue by artemisxrpg** [mart1nro/joycontrol#92]
Trying to look for ways to set multiplayer via parsec. Would it be possible to set up multiple controllers input and if so, what is the limit?

**Comment by Deltara3** [comment]
Yes, you can. I'm not sure of the limit, however, if you have two or more controllers plugged in it will mix inputs. Unless you're talking about multiple emulated controllers, then also yes. That requires multiple Bluetooth adapters though.
